### PR TITLE
TouchZoom and MouseWheelScroll

### PR DESCRIPTION
**Pull request by SveinLoken**

### CODE REVIEW

File: src/dom/DomEvent.js [hunk 1]
```diff
@@ -227,9 +227,10 @@ L.DomEvent = {
 
 	// Chrome on Win scrolls double the pixels as in other platforms (see #4538),
 	// and Firefox scrolls device pixels, not CSS pixels
-	_wheelPxFactor: (L.Browser.win && L.Browser.chrome) ? 2 :
-	                (L.Browser.gecko) ? window.devicePixelRatio :
-	                1,
+	_wheelPxFactor: ((L.Browser.win && L.Browser.chrome) || L.Browser.safari) ? 2 :
+					L.Browser.edge ? 3 :
```
Comment (IvanSanchez) [inline src/dom/DomEvent.js]: OK, so I think this is trying to address #4592

File: src/map/handler/Map.TouchZoom.js [hunk 2]
```diff
@@ -109,6 +109,15 @@ L.Map.TouchZoom = L.Handler.extend({
 		    .off(document, 'touchmove', this._onTouchMove)
 		    .off(document, 'touchend', this._onTouchEnd);
 
+		var snap = L.Browser.any3d ? this._map.options.zoomSnap : 1;
+		if (snap) {
```
Comment (IvanSanchez) [inline src/map/handler/Map.TouchZoom.js]: Why are you changing the touch zoom behaviour in the same pull request with a scrollwheel change? If there is an issue about touch-zoom snapping, do provide a bug number for it, and split the change into another commit & pull req.

File: src/dom/DomEvent.js (hunk 1 above)
Comment (SveinLoken) [inline src/dom/DomEvent.js]: Yes:
Chrome: 50
Firefox: 54
Safari: 120
Edge: 122.7
IE11: 60

File: src/map/handler/Map.TouchZoom.js (hunk 2 above)
Comment (SveinLoken) [inline src/map/handler/Map.TouchZoom.js]: Sorry! I am a newbie! Should I remove everything and create new pull requests?

Comment (IvanSanchez) [inline src/map/handler/Map.TouchZoom.js]: Yes. One pull request per bug or feature. This pull request has **3** different things:
* A change in `L.Browser.chrome` that is not really needed (Edge having 'chrome' in the UA string doesn't break anything - if it breaks, send a bug report explaining what is broken) - and BTW this happens only in Edge 14, not in Edge 12
* A possible fix for #4592
* A change of behaviour in touch-zoom snapping (which has had no discussion at all, open up a bug about why it's needed)

As a general rule, don't make several changes in several pieces of the software in one commit, unless the changes are very tightly coupled (this is not such a case).

Comment (SveinLoken) [inline src/map/handler/Map.TouchZoom.js]: In my test above edge will return a value if 2 because it match (L.Browser.win && L.Browser.chrome). So I think they are tightly coupled (agree?).
What to do now? Remove my pull request completely before I follow your general rule description?

Comment (IvanSanchez) [inline src/map/handler/Map.TouchZoom.js]: There's still no need to change `L.Browser.chrome`. Just check for edge first, then chrome, then ffx, then everything else.

Either remove this PR and create a new one, or amend it to remove the unneeded changes. That's up to you.

Comment (SveinLoken) [inline src/map/handler/Map.TouchZoom.js]: @IvanSanchez I think #4592 will solve the problem with edge. Sorry to disturbed you. Next time I know how to create a correct pull request.
